### DISCUSSION TITLE
Feature: Add CAS (and maybe SAML) as a authentication method

### DISCUSSION THREAD

**Discussion by jayteaftw** [langfuse/langfuse#2923] [Ideas]
### Describe the feature or potential improvement

Hi the organization I use only allows for CAS authentication, and I would like to be able to integrate this service into our cluster. And if that is possible, if we could also get SAML that would be fantastic.
Here are some resources explaining the protocols
[CAS overview](https://www.okta.com/identity-101/central-authentication-service/)
[CAS protocol](https://apereo.github.io/cas/6.6.x/protocol/CAS-Protocol.html)
[SAML overview](https://www.onelogin.com/learn/saml)

**Comment by marcklingen** [comment]
Langfuse uses next-auth/auth.js, there are some saml bridges as far as I know (my last research is from a while back). Let's track interest for this here, open for contributions on this
